recipes/octave/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = "4.2.1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: octave
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/octave/octave-{{ version }}.tar.gz
+  sha256: 80c28f6398576b50faca0e602defb9598d6f7308b0903724442c2a35a605333b
+  patches:
+    # patches taken from macports
+    # https://github.com/macports/macports-ports/tree/master/math/octave/files
+    # Link fails with undefined symbol ~file_stat
+    - patch-file-stat.cc.diff
+    # qscintilla2 library is not found without this
+    - patch-qscintilla2.diff
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Maybe we can to re-package an official binary on Windows. Not ideal but at least we will provide `octave` for all platforms.